helm/values.yaml

```diff
@@ -10,6 +10,9 @@ global:
     useLocalhost: false
     traceEnabled: true
     prometheusCollectEnabled: false
+    sqlServerUrl: ""
```

Comment:
it's better putting dummy values that SCREAM to the user that something is missing

otherwise... the cause of the error may be pretty invisible....

Reply:
Would you do it in the Dockerfile as well ?